tests/subsys/llext/hello_world/hello_world.c

```diff
@@ -11,15 +11,14 @@
  * a function.
  */
 
-/* Various build options should be documented here to generate the test elf for
- * each architecture.
- *
- * armv7-thumb: -mlong-call -mthumb -c -o hello_world_armv7_thumb.elf hello_world.c
- */
+#include <stdint.h>
 
 extern void printk(char *fmt, ...);
 
+static const uint32_t number = 42;
+
 extern void hello_world(void)
 {
 	printk("hello world\n");
+	printk("Math a number is %lu\n", number);
```

Comment:
Would you mind updating the sample documentation accordingly? In addition to simply updating the expected output feel free to add a note on why it is that there are two different printk's and what that demonstrate, if you think this can be useful information to the reader.